source/winVersion.py

```diff
@@ -26,3 +26,27 @@ def canRunVc2010Builds():
 UWP_OCR_DATA_PATH = os.path.expandvars(r"$windir\OCR")
 def isUwpOcrAvailable():
 	return os.path.isdir(UWP_OCR_DATA_PATH)
+
+def isAtLeastWin10(version=1507):
+	"""
+	Returns True if NVDA is running on at least the supplied version of Windows 10. If no argument is supplied, returns True for all public Windows 10 releases.
+	Note: this function will always return False for source copies of NVDA due to a Python bug.

```

Comment:
Can you specifically explain the 'version' argument in the docstring, in that this is a public release version and not a build version?